app/components/options_context/options_context.android.js

```diff
@@ -35,6 +40,19 @@ export default class OptionsContext extends PureComponent {
     };
 
     render() {
-        return null;
+        return (
+            <TouchableHighlight
+                onHideUnderlay={() => this.props.toggleSelected(false)}
```

Comment:
Somewhat unrelated but since we had our performance conversation. How do we feel about passing in functions as properties? If we're worried about re-renders then I know this creates a new function on each render call which would defeat the purpose of PureComponent. Thoughts on only using class methods and removing these where we can?